server/monitor.go

```diff
@@ -2005,68 +2012,121 @@ func (s *Server) accountInfo(accName string) (*AccountInfo, error) {
 	}
 	a.mu.RLock()
 	defer a.mu.RUnlock()
-	claim, _ := jwt.DecodeAccountClaims(a.claimJWT) // ignore error
+	var vrIssues []ExtVrIssues
+	claim, _ := jwt.DecodeAccountClaims(a.claimJWT) //ignore error
+	if claim != nil {
+		vr := jwt.ValidationResults{}
+		claim.Validate(&vr)
+		vrIssues = make([]ExtVrIssues, len(vr.Issues))
+		for i, v := range vr.Issues {
+			vrIssues[i] = ExtVrIssues{v.Description, v.Blocking, v.TimeCheck}
+		}
+	}
 	exports := []ExtExport{}
 	for k, v := range a.exports.services {
-		e := ExtExport{
-			Export: jwt.Export{
-				Subject:      jwt.Subject(k),
-				Type:         jwt.Service,
-				TokenReq:     v.tokenReq,
-				ResponseType: jwt.ResponseType(v.respType.String()),
-			},
-			ApprovedAccounts: []string{},
-		}
-		for name := range v.approved {
-			e.ApprovedAccounts = append(e.ApprovedAccounts, name)
+		var e ExtExport
```

Comment:
Would this be easier to read?
```
e := ExtExport{
   Subject: jwt.Subject(k),
   Type: jwt.Service,
}
if v != nil {
  e.TokenReq = v.tokenReq
  e.ResponseType = ..
  ...
}
```
Basically create the export with the fields that don't depend on `v` being nil or not, then complete in the `if v != nil` section.

Reply:
fixed, essentially creating the empty export, overwriting in case of v != nil